template/src/pages/form/Login/index.js

```diff
@@ -1,17 +1,19 @@
 import { useForm } from 'react-hook-form';
 import { zodResolver } from '@hookform/resolvers/zod';
 import { z } from 'zod';
-import { Link } from 'react-router-dom';
+import { Link, useHistory } from 'react-router-dom';
 
 import routesPaths from 'routes/routesPaths';
 import Input from 'components/form/Input';
 import useTranslation from 'hooks/useTranslation';
 
 import '../styles.css';
+import { useLoginMutation } from 'services/auth';
 
 const Login = () => {
   const t = useTranslation();
-
+  const [login, { isLoading, error }] = useLoginMutation();
+  const history = useHistory();
```

Comment:
WDYT about this suggestion? then you should replace the `history.push()` for just `push()`

```suggestion
  const {push} = useHistory();
```